tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -10,7 +10,76 @@ const util = require('util')
 let deletedElements
 let timeOfLastDeleteOperation = Date.now()
 let timeOfLastUploadOperation = Date.now()
-const { download } = require('../helpers/webdavHelper')
+const convert = require('xml-js')
+const { parts } = require('../helpers/path')
+const { download, propfind } = require('../helpers/webdavHelper')
+
+const getAllFilesFolders = function () {
```

Comment:
This function is telling a lie. Does it really return a list of files and folders? Please, honor the contract you try to set, and, please add documentation. :slightly_smiling_face: